submissions/TaniaBondarenko/popup2_task/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hooly-style Popup</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <!--Navigation panel with menu-button-->
```

Comment:
Remove this comment, please. It's redundant.
I recommend to watch this [video](https://www.youtube.com/watch?v=yhF7OmuIILc&ab_channel=JSConf). There are a lot of useful pieces of advice about the comments